### PR TITLE
Adding quantum dice

### PR DESCRIPTION
**Pull request by JRussellHuffman**
Beginning the process of publishing the quantum dice roller app.

### CODE REVIEW

File: content/experiments/quantum-dice.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Quantum Dice
+author: Russell Huffman
+description: A simple dice roller app using quantum computing to generate random numbers.
+image: https://github.com/JRussellHuffman/quantum-dice/blob/master/assets/header-image.pngs
```
Comment (delapuente) [inline content/experiments/quantum-dice.md]: This is not really an image but a GitHub page. To serve the actual image use [raw.githack.com](https://raw.githack.com/) to convert from a GitHub URL to the actual file URL. Use the production URL to replace the current one.

File: content/experiments/quantum-dice.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+to: /experiments/quantum-dice
+media:
+  - https://github.com/JRussellHuffman/quantum-dice/blob/master/assets/header-image.pngs
```
Comment (delapuente) [inline content/experiments/quantum-dice.md]: Same here. Be careful with the `s` in `pngs`. I think the extension is just `png`.

File: content/experiments/quantum-dice.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  - https://medium.com/design-ibm/truly-quantum-dice-cfe372f4c586
```
Comment (delapuente) [inline content/experiments/quantum-dice.md]: Cannot link to a medium post from media. This is intended for pictures and videos right now. Can you remove it, please.

**Comment by JRussellHuffman** [comment]
I really like the idea of adding a dropdown for available backends. Would be a good feature if I ever do a v2.

**Pull request merged**